src/cli/index.js

```diff
@@ -39,16 +39,16 @@ process.on("unhandledRejection", (reason, p) => {
     }
 });
 
-exports.run = () => {
-    const program = new Command();
+exports.run = (opts = {}) => {
+    const program = new Command(opts.cliName || "testplane");
 
     program.version(pkg.version).allowUnknownOption().option("-c, --config <path>", "path to configuration file");
 
     const configPath = preparseOption(program, "config");
     hermione = Hermione.create(configPath);
 
     program
-        .on("--help", () => logger.log(info.configOverriding))
+        .on("--help", () => logger.log(info.configOverriding(opts)))
```

Comment:
name `configOverriding` looks like it is a string.

If you change its type to `Function`, i suggest to rename it to something like `getConfigOverriding`, to emphasize it is a function

Reply:
My changes is optimized for future situation: just get rid of old staff.

I'm looking forward to delete opts here and in similar places.